Python/pywarpx/picmi.py

```diff
@@ -805,6 +807,13 @@ def initialize_inputs(self):
         for diagnostic in self.diagnostics:
             diagnostic.initialize_inputs()
 
+        if self.amr_checkpoint:
+            if os.path.exists(self.amr_checkpoint):
+                pywarpx.amr.restart = self.amr_checkpoint
+            else:
+                print(f'Could not find checkpoint {self.amr_checkpoint} !')
```

Comment:
The check if the file exists is not necessary here since the same check is already done in the C++ code during the restart. The simple statement `pywarpx.amr.restart = self.amr_restart` is sufficient, with neither check needed.